app/services/shopify_import/products/create.rb

```diff
@@ -6,9 +6,10 @@ def initialize(shopify_data_feed)
       end
 
       def save!
-        Spree::Product.transaction do
+        Spree::Product.transaction(isolation: :serializable) do
```

Comment:
This seems to be breaking the build on travis. Why do you need this level of isolation?